DESCRIPTION

```diff
@@ -23,7 +23,9 @@ LazyData: true
 Suggests: 
     knitr,
     rmarkdown,
-    testthat (>= 2.1.0)
+    testthat (>= 2.1.0),
+    pander,
+    devtools

```

Comment:
I don;t think this is correct. Devtools should not be a dependency as it is PURELY to develop a package. I'd say it's the same for pander

Reply:
You are right about devtools, it does not need to be in the vignette because it is not used inside the vignette, I just removed it. However, pander is used in the vignette, so it needs to be in the suggested section of namespace.
The documentation: http://r-pkgs.had.co.nz/check.html was quite helpful about the process:
"Checking for unstated dependencies in vignettes. As with tests, every package that you use in a vignette must be listed in the DESCRIPTION. If a package is used only for a vignette, and not elsewhere, make sure it’s listed in Suggests"